Source/Core/DolphinQt/Config/Graphics/GeneralWidget.cpp

```diff
@@ -176,7 +183,6 @@ void GeneralWidget::ConnectWidgets()
     m_custom_aspect_width->setHidden(!is_custom_aspect_ratio);
     m_custom_aspect_height->setHidden(!is_custom_aspect_ratio);
   });
-  m_aspect_combo->currentIndexChanged(m_aspect_combo->currentIndex());
```

Comment:
Instead of duplicating the lambda's logic in `CreateWidgets` I'd suggest making it a lambda variable here. You can pass the variable to `connect` and then call it directly to update the custom aspect visibility without the other side effects of currentIndexChanged.

Reply:
I thought about that. But I don't think the Connections function is a logical place for setting initial values.

Reply:
What about putting the logic in a separate function? Then you could call it from `CreateWidgets` without the code duplication.